src/category_theory/adjunction/basic.lean

```diff
@@ -208,11 +216,15 @@ def mk_of_unit_counit (adj : core_unit_counit F G) : F ⊣ G :=
   end },
   .. adj }
 
+/-- The adjunction between the identity functor on a category and itself. -/
 def id : 𝟭 C ⊣ 𝟭 C :=
 { hom_equiv := λ X Y, equiv.refl _,
   unit := 𝟙 _,
   counit := 𝟙 _ }
 
+/-- Satisfy the inhabited linter. -/
```

Comment:
You can satisfy the doc linter with real descriptions ;) this is no better than a `@[nolint doc_blame]`.

Reply:
This isn't for the doc linter - I don't think the doc linter complains about instances?

Reply:
Ah perhaps I misread this - are you saying that this should be a comment rather than a docstring? In that case, my mistake

Reply:
Right, if you make this a docstring, it better be useful (-;

Reply:
Amended to a comment!